src/webchat-ui/components/plugins/MessagePluginRenderer.tsx

```diff
@@ -80,10 +85,10 @@ export default ({ message, config, onSendMessage, plugins, isFullscreen, onSetFu
                     <MessageRow
                         key={key}
                         align={message.source === 'bot' ? 'left' : 'right'}
-                        className={message.source === 'bot' ? 'webchat-chat-message-bot': 'webchat-chat-message-user'}
+                        className={message.source === 'bot' ? regularMessageBot : regularMessageUser }
                     >
                         <Avatar src={avatarImg}
-                        className={message.source === 'bot'? 'webchat-chat-message-bot-avatar': 'webchat-chat-message-user-avatar'}
+                            className={message.source === 'bot' ? botAvatar : userAvatar }
```

Comment:
this is not part of the message plugin, should be `webchat-avatar` +  `user` or `bot`

Reply:
fixed